src/components/ChatList/elements/ChannelImage.style.ts

```diff
@@ -56,7 +53,7 @@ export const channelImageStyles = StyleSheet.create({
     height: dimen.normalizeDimen(24),
     borderRadius: dimen.normalizeDimen(12),
     borderWidth: 2,
-    borderColor: COLORS.white,
+    borderColor: colors.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor value has been changed. Make sure this new color (`colors.white`) is equivalent to the old one (`COLORS.white`). If it is not equivalent, this could lead to unexpected changes in the app's appearance.
```diff
-    borderColor: COLORS.white,
+    borderColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->